### PR TITLE
[2.26.x] fix npe in rtf

### PR DESCRIPTION
**Pull request by glenhein**
#### What does this PR do?

fix npe in rtf

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by glenhein** [comment]
build now

**Review by jlcsmith — APPROVED** [review]

File: catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/model/ExportCategory.java [hunk 1]
```diff
@@ -138,9 +138,12 @@ private ExportValue simpleValue(Attribute attribute) {
   }
 
   private static boolean isNonEmptyValue(Metacard metacard, String attrName) {
-    final AttributeDescriptor descriptor =
-        metacard.getMetacardType().getAttributeDescriptor(attrName);
+    AttributeDescriptor descriptor = metacard.getMetacardType().getAttributeDescriptor(attrName);
     final Attribute attribute = metacard.getAttribute(attrName);
+    if (descriptor == null) {
+      return attribute != null && StringUtils.isNotEmpty(attribute.getValue().toString());
```
Comment (jrnorth) [inline catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/model/ExportCategory.java:144]: We also need to check that `attribute.getValue()` is not null.

Comment (glenhein) [reply catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/model/ExportCategory.java:144]: I will add the check.

**Comment by glenhein** [comment]
build now

**Comment by glenhein** [comment]
build now

**Review by jrnorth — APPROVED** [review]

**Comment by alexabird** [comment]
Starting hero on this

**Comment by alexabird** [comment]
Hero looks good ✅

**Pull request merged**